Modules/posixmodule.c

```diff
@@ -2484,6 +2484,12 @@ posix_do_stat(PyObject *module, const char *function_name, path_t *path,
     STRUCT_STAT st;
     int result;
 
+#  if __has_feature(memory_sanitizer)
```

Comment:
It requires a check if `__has_feature` is defined

Reply:
pyport.h defines `_Py_MEMORY_SANITIZER` for us, taking care of all of the compiler/platform conditionals around this.